### DISCUSSION TITLE
Create update site from product defintion

### DISCUSSION THREAD

**Discussion by sebthom** [eclipse-tycho/tycho#338] [Q&A] [answered]
Is it possible to create an update site from all the dependencies of a product definition using a `<packaging>eclipse-repository</packaging>` Maven module?

I'd like to provide an update-site that can be used to fully update a product. Manually maintaining a category.xml does not feel DRY.

**Comment by laeubi** [comment, marked as answer]
When building the product you get a `target/repository` folder as a result that includes an update-site specific for that product.

**Comment by sebthom** [comment]
Ah nice! @laeubi thank you!!

**Comment by laeubi** [comment]
By the way, we have enabled discussions for this repository, questions regarding the usage of tycho are fitting better there (I'll convert the ticket) and you can accept an answer there if it solves your issue to help other users to find a solution.
